_monitoring-plugins/alerting/monitors.md

```diff
@@ -490,14 +490,14 @@ To create a cluster metrics monitor:
 
 Trigger conditions use responses from the following APIs. Most APIs that can be used to monitor cluster status support path parameters as described in their documentation (e.g., comma-separated lists of index names). However, they do not support query parameters.
 
-1. [_cluster/health]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-health/)
-2. [_cluster/stats]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-stats/)
-3. [_cluster/settings]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/)
+1. [_cluster/health]({{site.url}}{{site.baseurl}}/api-reference/cluster-health/)
```

Comment:
These are paths or endpoints, not "APIs." Recommend either rephrasing the preceding paragraph to reference "API endpoints" or "API paths." Otherwise, each of these links should be reformatted like this:

`[Cluster health]({{site.url}}{{site.baseurl}}/api-reference/cluster-health/)`